docs/clearml_agent/clearml_agent_env_var.md

```diff
@@ -47,4 +48,5 @@ but can be overridden by command-line arguments.
 |**CLEARML_AGENT_CHILD_AGENTS_COUNT_CMD** | Provide an alternate bash command to list child agents while working in services mode                                                                                                                                                           |
 |**CLEARML_AGENT_SKIP_PIP_VENV_INSTALL** | Instead of creating a new virtual environment inheriting from the system packages, use an existing virtual environment and install missing packages directly to it. Specify the python binary of the existing virtual environment. For example: `CLEARML_AGENT_SKIP_PIP_VENV_INSTALL=/home/venv/bin/python`                                                                                                                                                             |
 |**CLEARML_AGENT_SKIP_PYTHON_ENV_INSTALL** | If set to `1`, the agent will not install any required python packages and will just use the preexisting python environment to run the task.                                                                                                                                            |
-|**CLEARML_AGENT_VENV_CACHE_PATH** | Overrides venv cache folder configuration                                                                                                                                                                                                       |
\ No newline at end of file
+|**CLEARML_AGENT_VENV_CACHE_PATH** | Overrides venv cache folder configuration                                                                                                                                                                                                       |
+|**CLEARML_MULTI_NODE_SINGLE_TASK**| Control how multi-node resource monitoring is reported. The options are: <ul><li>`-1` - Only master node's (rank zero) console/resources is reported</li><li>`1` - Graph per node i.e. machine/GPU graph for every node (console reported with prefix RANK x)</li><li>`2` - Series per node under a unified machine resource graph, graph per type of resource e.g. CPU/GPU utilization (console reported with prefix RANK x)</li></ul>|
```

Comment:
@pollfly
* "console/resources is reported" -> "console/resources are reported"?
* "console reported with prefix RANK x" -> "console output prefixed with RANK"